models/cfp.py

```diff
@@ -792,6 +803,55 @@ class InstallationProposal(Proposal):
     funds = db.Column(db.String, nullable=True)
 
 
+class LightningTalkProposal(Proposal):
+    __mapper_args__ = {"polymorphic_identity": "lightning"}
+    human_type = HUMAN_CFP_TYPES["lightning"]
+    slide_link = db.Column(db.String, nullable=True)
+    # This should probably use allowed_times or something but that has a very
```

Comment:
I think this is the right way, so we can scratch this comment to prevent later doubt :)